stable/datadog/templates/daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.datadog.apiKey }}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```

Comment:
FYI @gtaylor - you add a flag to install the kube-state-metrics. Here's the deployment from the coreos prometheus operator.

https://coreos.com/operators/prometheus/latest/exporters.yaml